pyeudiw/sd_jwt/__init__.py

```diff
@@ -0,0 +1,67 @@
+from io import StringIO
+from sd_jwt.issuer import SDJWTIssuer
+from sd_jwt.verifier import SDJWTVerifier
+
+from sd_jwt.utils.demo_utils import get_jwk
+from sd_jwt.utils.formatting import textwrap_json
+from sd_jwt.utils.yaml_specification import _yaml_load_specification
+
+from pyeudiw.tools.utils import iat_now
+from pyeudiw.jwk import JWK
+
+def _adapt_keys(settings: dict, issuer_key: JWK, holder_key: JWK, kty: str = "EC", key_size: int = 256):
+    keys = {
+        "key_size": key_size,
+        "kty": kty,
+        "issuer_key": issuer_key.as_dict(),
+        "holder_key": holder_key.as_dict()
+    }
+    
+    return get_jwk(keys, settings["no_randomness"], None)
+
+def load_specification_from_yaml_string(yaml_specification: str):
+    return _yaml_load_specification(StringIO(yaml_specification))
+
+def issue_sd_jwt(specification: dict, settings: dict, issuer_key: JWK, holder_key: JWK) -> str:    
+    claims = {
+        "iss": settings["issuer"],
+        "iat": iat_now(),
+        "exp": iat_now() + (settings["default_exp"] * 60)  # in seconds
```

Comment:
please add a function in the tools to get the default_exp instead of using * 60 everytime  in the code